.github/workflows/release.yml

```diff
@@ -93,13 +98,28 @@ jobs:
           profile: minimal
           target: ${{ matrix.target }}
 
+      - name: Setup | Install cargo-wix [Windows]
```

Comment:
A passing thought — not something we have to address in this PR:
Our release-time build process is getting pretty complex with building installers and notarizing, which leads to a lot of unexpected failures at the time of release.

For this, in particular, I think there'd be value in adding this to CI. Maybe not on every commit, like the rest of our suite, but perhaps on every push to `master`.

Reply:
Done.